src/templates/Navigator/index.js

```diff
@@ -53,16 +52,9 @@ function Navigator(props) {
                   steps={stepProgressData}
                 />
 
-                <Sticky
```

Comment:
Did you mean to remove the sticky? Or did you only take it out for the symlink and forget to put it back?